styles.css

```diff
@@ -1,146 +1,296 @@
+section {
+  position: relative;
+  top: 48px;
+  border-radius: 0 0 0 10%;
+  padding-left: 24px;
+  padding-right: 24px;
 }
 
-.navbar a {
+a {
   text-decoration: none;
+  color: #344563;
 }
 
-.logo {
+header {
+  background: #fff;
+}
+
+li {
+  list-style: none;
+}
+
+.mobile-portfolio {
+  position: absolute;
+  left: 0;
+  right: 0;
   display: flex;
-  flex-direction: row;
+  width: 100%;
+  height: 50px;
+  justify-content: space-between;
   align-items: center;
-  padding: 10px 12px;
-  position: absolute;
-  width: 100px;
-  height: 40px;
-  left: 12px;
-  top: calc(50% - 40px / 2);
+  padding: 14px 24px;
 }
 
-.logo a {
-  width: 76px;
-  height: 20px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
+.logo {
   font-weight: 700;
   font-size: 18px;
-  line-height: 20px;
-  display: flex;
-  align-items: center;
   text-align: center;
   color: #6070ff;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
 }
 
-.menu {
-  display: flex;
-  flex-direction: row;
-  align-items: flex-start;
-  padding: 8px;
-  position: absolute;
-  width: 40px;
-  height: 40px;
-  right: 24px;
-  top: calc(50% - 40px / 2);
-  border-radius: 8px;
-}
-
-.bx {
-  position: relative;
-  right: 10.67%;
-  top: 32%;
-  bottom: 50%;
-  font-size: 30px;
-}
-
-.headline {
-  background-image: url("header-shapes mobile.svg");
+.first-page {
+  min-width: 375px;
+  padding-top: 80px;
+  padding-bottom: 80px;
+  background: #fff;
+  background-image: url("./image/bg.svg");
   background-repeat: no-repeat;
-  display: flex;
-  flex-direction: column;
-  align-items: flex-start;
-  padding: 114px 24px;
-  gap: 12px;
-  position: absolute;
-  width: 375px;
-  height: 604px;
-  left: calc(50% - 375px / 2);
-  top: 60px;
+  background-size: calc(100%);
 }
 
 .name {
-  width: 327px;
-  height: 150px;
   font-family: 'Poppins', sans-serif;
   font-style: normal;
-  font-weight: 700;
-  font-size: 40px;
+  font-weight: 900;
+  font-size: 50px;
   line-height: 52px;
   display: flex;
   align-items: center;
   color: #172b4d;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
 }
 
 .text {
-  width: 327px;
-  height: 146px;
   font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
   color: #344563;
-  flex: none;
-  order: 1;
-  flex-grow: 0;
+  margin-right: 10px;
+  background: none;
 }
 
-.media {
-  width: 130px;
-  height: 24px;
+.connect {
   font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 16px;
   line-height: 24px;
   color: #7f8cff;
+  text-decoration: none;
+  margin-top: 30px;
+}
+
+.social-icons {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 10px;
+  margin-left: -40px;
+}
+
+.work {
+  background: #e5e5e5;
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 10px;
+}
+
+.project {
+  background: #fff;
+  border: 0.2rem solid #e5e5e5;
+  border-radius: 20px;
+  flex-direction: row;
+}
+
```

Comment:
- [ ] I think it would be better if you add a little more spacing between your header and work section, so that we can have the design according to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=JXWrOAMbfgkr3ywB-0). I know you can do that 😃💪

### **_Your design previously 🔽_**
![image](https://user-images.githubusercontent.com/106119105/212492131-24f9f8a2-e3e2-4d73-b4dc-71cac7c8f375.png)
### **_Your design now 🔽_**
![image](https://user-images.githubusercontent.com/106119105/212492141-c0c8d6d1-fd2b-4657-890a-1c9319a34ff2.png)
### **_figma template 🔽_**
![image](https://user-images.githubusercontent.com/106119105/212492167-415345fd-73ef-4e68-b8b3-8d8cccd54ce6.png)

- [ ]  Please follow the same for other cards as well 😃